keras/src/ops/nn.py

```diff
@@ -818,6 +818,48 @@ def hard_shrink(x, threshold=0.5):
     return backend.nn.hard_shrink(x, threshold)
 
 
+class Threshold(Operation):
+    def __init__(self, threshold_value, value):
+        super().__init__()
+        self.threshold_value = threshold_value
+        self.value = value
+
+    def call(self, x):
+        return backend.nn.threshold(x, self.threshold_value, self.value)
+
+    def compute_output_spec(self, x):
+        return KerasTensor(x.shape, dtype=x.dtype)
+
+
+@keras_export(["keras.ops.threshold", "keras.ops.nn.threshold"])
+def threshold(x, threshold_value, value):
```

Comment:
Let's adjust the arg names a bit:

`threshold_value` -> `threshold`, to be consistent with other activations like ReLU
`value` -> `default_value`